autoscheduler/frontend/src/utils/sortMeetingFunction.tsx

```diff
@@ -0,0 +1,13 @@
+import Meeting from '../types/Meeting';
+
+// sorts meeetings based on meeting type
+const sortMeeting = (a: Meeting, b: Meeting): number => {
+  if (a.meetingType > b.meetingType) {
+    return 1;
+  }
+  if (a.meetingType < b.meetingType) {
+    return -1;
+  }
+  return 0;
```

Comment:
I _think_ this can be done with [string.localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare), so you should be able to change this to `return a.meetingType.localeCompare(b.meetingType)`. Might have to flip a & b though. Other than this I think it'll be good to go

Reply:
Actually I think the way he has it is the way we should be doing this. This is comparing the enum values rather than the string values, which means we can reorder the enum values in order to define the order they show up in, and let important meetings like LEC/LAB show up before EXAM meetings. It's also a lot more flexible because if we decide to change it later we can just reorder the enum

Reply:
Why not just return `a.meetingType - b.meetingType`?